project-settings/other-project-settings/security-tab/scim-integration-with-okta.md

```diff
@@ -51,7 +60,7 @@ You may then proceed to configure your app:
 
 Create mapping rules to automatically provide values to mandatory parameters `teams`, `role`, and `permissionLevel`, and optionally `tags`. If you don’t create mapping rules, you will have to provide these values manually for each user provisioned.
 
-* permissionLevel (string): should match any of “Admin”, “Developer”, “Editor”, or “User”.
+* permissionLevel (string): should match any of `admin`, `developer`, `editor`, or `user`.
```

Comment:
👀 We don't support both syntaxe?

Reply:
Yes we do, but lowercase is what we actually expect, and this is for consistency